### ISSUE TITLE
Microphone support?

### ISSUE THREAD

**Issue by Daxode** [steward-fu/nds#44]
Anyways to map a Miyoo Mini+ key to the mic? (Needed to play Spectrobes!)

**Comment by steward-fu** [comment]
Yes, the DraStic emulator supports MIC by default. I will take a look at this and then update it after complete. Thanks

**Comment by Ichibruh** [comment]
Loving the ability to play NDS Zelda - Phantom Hourglass, would love to have the ability to generate static noise or use a microphone sample file to complete the story! You're awesome 😎

**Comment by Schabovy** [comment]
Yes, adding static noice functionality would allow to play many more games such as Zelda, Resident Evil Deadly Silence, Mario & Luigi etc. That would be a great thing!

**Comment by Johny-LPM** [comment]
Indeed, this would be a really awesome feature that'd expand game support!

**Comment by neilswann80** [comment]
Any progress on this?  I'm stuck at the entrance to the Fire Temple in Phantom Hourglass.

**Comment by steward-fu** [comment]
Long time not touch on this repo, ha ha, I think I will complete this feature soon.

**Comment by Ichibruh** [comment]
> Long time not touch on this repo, ha ha, I think I will complete this feature soon.

You're a G, respect the effort! Thanks for checking back on this repo ;)

**Comment by Frostoman** [comment]
I don't know if I can write about this problem here, but on R36S with ArkOS or Amberelec the microphone doesn't work either. I assigned a button in the menu, but it does nothing

**Comment by nathacks** [comment]
What would be great is to be able to adjust the microphone sensitivity directly in a game, and assign two keys to increase or decrease it.

**Comment by bqcuong** [comment]
Hi @steward-fu, any updates on this feature?

**Comment by steward-fu** [comment]
The micphone is ready for test on miyoo mini (plus) handheld. Can anyone help test ?

**Comment by neilswann80** [comment]
> The micphone is ready for test on miyoo mini (plus) handheld. Can anyone help test ?

What do you need me to do?

**Comment by steward-fu** [comment]
test build: https://drive.google.com/file/d/1IYxS8Gvnicu--sXophNaVmQ4ERgyRDOw/view?usp=sharing
MENU+UP: toggle microphone
